manifest.json

```diff
@@ -7,12 +7,12 @@
   "homepage_url": "https://github.com/CanopyTax/single-spa-inspector",
   "description": "A devtools panel for single-spa applications",
   "browser_action": {
-    "default_icon": "./logo-white-bgblue.svg",
+    "default_icon": "./logo-white-bgblue.png",
     "default_title": "single-spa Inspector"
   },
   "icons": {
-    "48": "./logo-white-bgblue.svg",
-    "96": "./logo-white-bgblue.svg"
+    "48": "./logo-white-bgblue.png",
+    "96": "./logo-white-bgblue.png"
```

Comment:
I only made the png have 96px dimensions, since it seemed like that was all that was needed? I tested this in firefox and it looked fine both in the main firefox navbar and in the dev tools tab. If we need a bigger png let me know